Source/EasyNetQ/IAdvancedBus.cs

```diff
@@ -15,14 +16,14 @@ namespace EasyNetQ
     public interface IAdvancedBus : IDisposable
     {
         /// <summary>
-        /// True if the bus is connected, False if it is not.
+        /// True if a connection of the given type is established. 
         /// </summary>
-        bool IsConnected { get; }
+        bool IsConnected(PersistentConnectionType type);
```

Comment:
I would add also the property `IsConnected` that checks both connections

Reply:
I'm a bit concerned about adding this: in a simple app when only a producer is used, IsConnected will return False.